api/src/main/java/org/datacleaner/descriptors/DescriptorProvider.java

```diff
@@ -46,12 +46,6 @@
 public interface DescriptorProvider {
 
     /**
-     * @return a collection of {@link DescriptorProviderStatus} describing
-     *         current status.
-     */
-    public Collection<DescriptorProviderStatus> getStatus();
-
-    /**
```

Comment:
That's a nope. This is part of the public API, and we don't remove public API methods without warning.